lua/nvim-tree.lua

```diff
@@ -633,6 +648,12 @@ local DEFAULT_OPTS = { -- BEGIN_DEFAULT_OPTS
     show_on_open_dirs = true,
     timeout = 400,
   },
+  modified = {
+    enable = false,
+    show_on_dirs = true,
+    show_on_open_dirs = false,
+    debounce_delay = 50,
```

Comment:
Could you please implement #1836 in this PR.

legacy.lua `refactored` can be used to silently migrate `update_focused_file.debounce_delay`

Reply:
Sure, I'll take it look.

Reply:
Actually do you mind if I remove `modified.debounce_delay` for now and do #1836 in another PR? I'm not too comfortable making this already pretty big PR any bigger.

Reply:
> Actually do you mind if I remove `modified.debounce_delay` for now and do #1836 in another PR? I'm not too comfortable making this already pretty big PR any bigger.

No worries. That is sensible.